src/main/kotlin/UpdateTaskHolder.kt

```diff
@@ -73,13 +78,42 @@ object UpdateTaskHolder {
 
     //This class does nothing
     class DummyTask: CancellableTask {
+        override fun createToReschedule(client: BitbucketClient): CancellableTask {
+            return DummyTask()
```

Comment:
Previous behaviour: if user changed plugin's Settings, a task was rescheduled and started to fail because user didn't enter a password yet.

Now DummyTask creates DummyTask if it is reschedued and creates new real task only when user enters a password.